docs/changelogs/CHANGELOG-2.22.md

```diff
@@ -0,0 +1,481 @@
+- Fix wrong quota filtering when `VirtualMachineInstancePreset.spec.cpu` has no quantity but only other fields ([#11046](https://github.com/kubermatic/kubermatic/pull/11046))
+- Add support for ToplogySpreadConstraint for Kubevirt VM ([#11114](https://github.com/kubermatic/kubermatic/pull/11114))
+- Add support for instancetype/preference ([#11182](https://github.com/kubermatic/kubermatic/pull/11182))
+- Update KubeVirt CCM to v0.4.0 and kubevirt.io/api to v0.58.0 ([#11249](https://github.com/kubermatic/kubermatic/pull/11249))
+- Switch StorageClasses init configuration from annotation to DC ([#11716](https://github.com/kubermatic/kubermatic/pull/11716))
+- Update KubeVirt DC: add additional network policies configuration ([#11659](https://github.com/kubermatic/kubermatic/pull/11659))
+- Update seed and cluster to config StorageClasses from seed ([#11701](https://github.com/kubermatic/kubermatic/pull/11701))
+- Graceful user cluster workload eviction in case of bare-metal infra node draining ([#11588](https://github.com/kubermatic/kubermatic/pull/11588))
+- Split CSI driver deployment between user and infra cluster and remove user cluster CSI access to KubeVirt cluster ([#11370](https://github.com/kubermatic/kubermatic/pull/11370))
+- Fix KubeVirt LB issue (wrong custer-isolation netpol): LB not accessible from outside user-cluster ([#11640](https://github.com/kubermatic/kubermatic/pull/11640))
+- Change CustomNetworkPolicies type (extract name) ([#11666](https://github.com/kubermatic/kubermatic/pull/11666))
+- Fix missing ccmClusterName feature in KubeVirt clusters upgraded to KKP 2.22 ([#11844](https://github.com/kubermatic/kubermatic/pull/11844))
+
+#### KubeOne Cluster Support
+
+- Update KubeOne image to v1.5.3 for KubeOne external cluster controller ([#11388](https://github.com/kubermatic/kubermatic/pull/11388))
+- Add separate list and details pages for KubeOne clusters ([#5412](https://github.com/kubermatic/dashboard/pull/5412))
+- Add wizard to import KubeOne clusters ([#5362](https://github.com/kubermatic/dashboard/pull/5362))
+- Add support to import AWS KubeOne clusters ([#5362](https://github.com/kubermatic/dashboard/pull/5362))
+- Add support to import Azure KubeOne clusters ([#5488](https://github.com/kubermatic/dashboard/pull/5488))
+- Add support to import GCP KubeOne clusters ([#5460](https://github.com/kubermatic/dashboard/pull/5460))
+- Add support to migrate KubeOne cluster container runtime ([#5499](https://github.com/kubermatic/dashboard/pull/5499))
+- Add support to select preset in credentials step of KubeOne wizard ([#5504](https://github.com/kubermatic/dashboard/pull/5504))
+- Add support to upgrade KubeOne machine deployment version ([#5561](https://github.com/kubermatic/dashboard/pull/5561))
+
+#### Web Terminal
+
+- Add Web terminal support for user clusters ([#4492](https://github.com/kubermatic/dashboard/pull/4492))
```

Comment:
`..... This allows users to connect to their clusters from the KKP dashboard, using only their browser.`

Reply:
Added, thanks!